static/app/components/sidebar/index.tsx

```diff
@@ -586,13 +615,21 @@ function Sidebar() {
   );
 
   return (
-    <SidebarWrapper aria-label={t('Primary Navigation')} collapsed={collapsed}>
+    <SidebarWrapper
+      aria-label={t('Primary Navigation')}
+      collapsed={collapsed}
+      hasNewNav={hasNewNav}
```

Comment:
Rather than passing this property to each component, could we use a context to share the state between each component?

Reply:
since this is just a feature flag i'd rather keep it simple and just pass the state rather than create a context
thankfully we're only going 1 layer deep here so i think context might be a little over kill